app/views/pupils/schools/show.html.erb

```diff
@@ -1,53 +1,71 @@
 <% content_for :page_title do %><%= @school.name %><% end %>
 
-<% if  @equivalence %>
-  <%= render 'equivalence', equivalence_content: @equivalence_content %>
+<% if !@equivalences_content.empty? %>
```

Comment:
@equivalences_content.present? would that also work here?

Reply:
Updated